script/BaseDeployCore.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { CreateXScript } from "createx-forge/script/CreateXScript.sol";
+import { console2 } from "forge-std/console2.sol";
+
+contract BaseDeployCore is CreateXScript {
```

Comment:
Let's call this `BaseDeploy` or `BaseBalmyDeploy`, since it doesn't have mucho to do with "core"